common/test/acceptance/tests/discussion/test_discussion_management.py

```diff
@@ -357,11 +357,72 @@ def test_not_divided_hides_discussion_topics(self):
 
     def test_disabling_cohorts(self):
         """
-        Test that disabling cohorts hides the cohort division scheme iff it is not the selected scheme
+        Test that the discussions management tab hides when there is <= 1 enrollment track, the Cohort division scheme
+         is not selected, and cohorts are disabled.
         (even without reloading the page).
         """
-        # TODO: will be added as part of AJ's work.
-        pass
+        self.disable_cohorting(self.course_fixture)
+        self.instructor_dashboard_page.visit()
+        self.assertFalse(self.instructor_dashboard_page.is_discussion_management_visible())
+
+    def test_disabling_cohorts_while_selected(self):
+        """
+        Test that disabling cohorts does not hide the discussion tab when there is more than one enrollment track.
+        Also that the division scheme for cohorts is visible iff it was selected.
+        (even without reloading the page).
+        """
+        add_enrollment_course_modes(self.browser, self.course_id, ['audit', 'verified'])
+
+        # Verify that the tab is visible
+        self.instructor_dashboard_page.visit()
+        self.assertTrue(self.instructor_dashboard_page.is_discussion_management_visible())
+        self.view_discussion_management_page()
+        self.discussion_management_page.select_division_scheme(self.discussion_management_page.COHORT_SCHEME)
+        self.disable_cohorting(self.course_fixture)
+        self.assertTrue(self.instructor_dashboard_page.is_discussion_management_visible())
+
+        # Go to Discussions tab and ensure that the correct scheme options are visible
+        self.view_discussion_management_page()
+        self.assertTrue(
```

Comment:
You can leave out testing that the enrollment track option is present. Other test points do that.